cts/CTStests.py

```diff
@@ -1412,6 +1412,22 @@ def __call__(self, node):
             tmpPats, self.Env["DeadTime"] + self.Env["StableTime"] + self.Env["StartTime"])
         watch.setwatch()
 
+        if self.CM.Env["have_systemd"]:
+            temporary_corosync_service = """
+[Unit]
+Description=Temporary Corosync Cluster Engine to delay restarting the service
+
+[Service]
+ExecStart=/usr/share/corosync/corosync start
+ExecStop=/usr/share/corosync/corosync stop
+Type=forking
+Restart=always
+RestartSec=70
+"""
+            os.system("cat <<-END >/tmp/corosync.service\n%s\nEND" % (temporary_corosync_service))
+            self.CM.install_helper("corosync.service", destdir="/run/systemd/system/", sourcedir="/tmp")
```

Comment:
IIUC, the usual convention would be create a file for example as:

/run/systemd/system/corosync.service.d/Restart.conf

with the only necessary piece configuration that is expected to override the default ones like:

[Service]                                                                                               │~                                                                                                        
Restart=no

And guess it'd be alright to directly create the file rather than installing it later on.

Reply:
Agreed about the override content, plus I'm not sure whether we need the override for corosync, pacemaker, or both (since pacemaker has a Requires for corosync). I'd name the file cts.conf to avoid any possible name collisions.

Reply:
This is alittle tricky because of the systemctl. When setting the Restart=no, the pacemaker will immediately recover the corosync. However, when setting

[Service]
Restart=always
RestartSec=70

the systemctl will recover the corosync only after 70 seconds.